mlflow/utils/cli_args.py

```diff
@@ -43,3 +43,10 @@
 # We use None to disambiguate manually selecting "4"
 WORKERS = click.option("--workers", "-w", default=None,
                        help="Number of gunicorn worker processes to handle requests (default: 4).")
+
+# default disable
+CONFIG = click.option("--config", "-c", default=None, metavar="FILE",
```

Comment:
Maybe we can call this `--gunicorn-config`? We may not use gunicorn forever, so would be good fo `serve` not to assume it.